src/styles/blocks/product.scss

```diff
@@ -0,0 +1,167 @@
+.product {
+  &--catalog {
+    padding-block: 50px;
+    padding-inline: 40px;
+    display: grid;
+    grid-template-rows: 2;
+    grid-template-columns: repeat(1, 200px);
+    row-gap: 46px;
+    column-gap: 48px;
+    justify-content: center;
+
+    @media (min-width: 488px) {
+      grid-template-columns: repeat(2, 200px);
+    }
+
+    @media (min-width: 768px) {
+      grid-template-columns: repeat(3, 200px);
+    }
+
+    @media (min-width: 1024px) {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+
+  &__card {
```

Comment:
add padding for all card sides to avoid margin-left/margin-right for inner elements

Reply:
<img width="633" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/70220667/03aefb7e-c324-4051-aa10-b9195e15f25a">